### PR TITLE
tests: fix lxd test which hangs on restore

### PR DESCRIPTION
**Pull request by sergiocazzolato**
The problem seems to be that the init system in the container isn’t
performing a clean shutdown when signaled by LXD. This conclusion comes
from the fact that there is not more hangs when we use --force.

### CODE REVIEW

File: tests/main/lxd/task.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ restore: |
         exit
     fi
 
-    lxd.lxc stop my-ubuntu
+    lxd.lxc stop my-ubuntu --force
```
[the remark this answers is not in the document]
Comment (sergiocazzolato) [reply tests/main/lxd/task.yaml]: Well, I ran 400 times the test and in all the executions the snap could be removed without any error. I am still running it to see if there is any problem, but so far 100% sucess.

[the remark this answers is not in the document]
Comment (zyga) [reply tests/main/lxd/task.yaml]: Did we get to the bottom of what was happening? I don't mind using --force but I don't feel confident this is not going to bite us down the line.

**Comment by sergiocazzolato** [comment]
@zyga still debugging it, the idea of this PR is to unblock most of the branches which are in red, and in parallel I am researching what could be the real cause of this problem

**Comment by bboozzoo** [comment]
It may be the case that the fault is located elsewhere. Looking at the test we just install whatever lxd revision is available in the store, launch a basic container (which pulls some, currently available rootfs version), install snapd inside and tear it down. A handfull of moving parts, with some not under our control.

**Pull request merged**